README.md

```diff
@@ -17,18 +17,15 @@ The [official CocoaPods website](http://cocoapods.org) has detailed instructions
 The following lines in a Podfile will install the SDK for an iOS app with a watch OS 2 extension: 
 
 ```
-source 'https://github.com/openwhisk/openwhisk-podspecs.git'
-
+install! 'cocoapods', :deterministic_uuids => false
 use_frameworks!
 
 target 'MyApp' do
-     platform :ios, '9.0'
      pod 'OpenWhisk'
 end
 
 target 'MyApp WatchKit Extension' do 
-     platform :watchos, '2.0'
-     pod 'OpenWhiskWatch'
+     pod 'OpenWhisk'
```

Comment:
Using `:git => 'https://github.com/openwhisk/openwhisk-client-swift.git'` is good for the repo in terms of development/contrbutors, but should not be the default for normal folks just trying to use it and not necessary contribute, I think folks should stick to versioning and not point to master of the repo.

I think in the README.md we have the Podfile example in sync with the version tag using `, :tag => '0.1.7'`
```
pod 'OpenWhisk', :git => 'https://github.com/openwhisk/openwhisk-client-swift.git', :tag => '0.1.7'
```

And of course we can have a note for others about removing the `:tag` property